controller/api/destination/server.go

```diff
@@ -100,6 +102,12 @@ func (s *server) Get(dest *pb.GetDestination, stream pb.Destination_GetServer) e
 		log,
 	)
 
+	var token contextToken
+	b := dest.GetContextToken()
+	if err := json.Unmarshal([]byte(b), &token); err != nil {
+		log.Debugf("Error: %v", err)
```

Comment:
This is a serious error so we should log it at error level.  We should also either return the error (causing the API request to fail) or explicitly set the contextToken to a default empty state.